packages/e2e/features/ui/components/authenticator/sign-in-with-email.feature

```diff
@@ -32,6 +32,8 @@ Feature: Sign In with Email
     And I click the "Sign In" button
     Then I see "Sign out"
 
+  # FORCE_CHANGE_PASSWORD tests are skipped as the temporary passwords used for these
```

Comment:
What state do these go into when they expire?

Reply:
They still have a `FORCE_CHANGE_PASSWORD` status in Cognito, and we receive the `remoteError` "Temporary password has expired and must be reset by an administrator."